src/FSharpLint.Core/Text.resx

```diff
@@ -243,6 +243,9 @@
   <data name="RulesCanBeReplacedWithComposition" xml:space="preserve">
     <value>Lambda may be able to be replaced with composition. e.g. `fun x -&gt; x |&gt; isValid |&gt; not` could be replaced with `isValid &gt;&gt; not`.</value>
   </data>
+  <data name="RulesUsedUnderscorePrefixedElements" xml:space="preserve">
+    <value>Please avoid using underscore-prefixed elements for the elements that are being used in the code.</value>
```

Comment:
@realmarv there's no other message that starts with "Please" in this file so even if being polite is nice, let's be consistent here

Reply:
Done